stories/module-analytics-setup.stories.js

```diff
@@ -58,6 +60,61 @@ function Setup( props ) {
 	);
 }
 
+function makeGtmPropertyStory( { permission, useExistingTag = false } ) {
```

Comment:
Let's not duplicate utilities between stories. If we end up needing it in both places, let's create a better place for this to stay. I realize these render separate components in the end, but everything else here is basically duplicated and could be abstracted.

Reply:
Updated.